python-aexpect.spec

```diff
@@ -40,7 +40,11 @@ Source0: https://github.com/avocado-framework/%{srcname}/archive/%{commit}.tar.g
 
 BuildArch: noarch
 Requires: python
-BuildRequires: python, python-setuptools
+BuildRequires: python
+
+%if 0%{?rhel}
+BuildRequires: python-setuptools
```

Comment:
Well it is required and the fact that on some Fedora versions they are already installed doesn't mean we should not list them (especially when we need conditional to include them on RHEL). I don't think this commit should be included (nor in downstream).

Reply:
I've actually tried this, and it works as intended.  The RPM build environment guaranteed by `mock` includes a number of packages, including say `gcc`, so there's no need to set explicit (very very common) `BuildRequires`.  The list of packages (or group of packages) change from distro to distro, and from version to version.

So, to make it clear, this is not masked by packages already installed in a given system, it's safe because it will be present in the minimal environment set up by mock when building packages.

Reply:
I'm not strongly against, I'm just pointing out that it's not everywhere (RHEL) and Fedora is not the only RPM-based distro. To me it'd make sense to simply list it. But again, not a strong opinion here.